packages/riipen-ui/src/components/Button.jsx

```diff
@@ -400,6 +401,88 @@ class Button extends React.Component {
             border-color: ${theme.palette.negative.main};
           }
 
+          .inverted {
+            background-color: white;
+            color: ${theme.palette.text.primary};
+            overflow: hidden;
+            position: relative;
+          }
+          .inverted::after {
+            border-radius: ${theme.shape.borderRadius.md};
+            border: 1px solid rgba(0, 0, 0, 0.23);
+            bottom: 0;
+            content: "";
+            left: 0;
+            position: absolute;
+            right: 0;
+            top: 0;
+          }
+          .inverted:hover::before {
+            background-color: ${theme.palette.grey[600]};
+            bottom: 0;
+            content: "";
+            left: 0;
+            opacity: 0.1;
+            position: absolute;
+            right: 0;
+            top: 0;
+          }
+
+          .inverted-primary {
+            color: ${theme.palette.primary.main};
+          }
+          .inverted-primary:hover::before {
+            background-color: ${theme.palette.primary.main};
+          }
+          .inverted-primary::after {
+            border-color: ${theme.palette.primary.main};
+          }
+          .inverted-secondary {
+            color: ${theme.palette.secondary.main};
+          }
+          .inverted-secondary:hover::before {
+            background-color: ${theme.palette.secondary.main};
+          }
+          .inverted-secondary::after {
+            border-color: ${theme.palette.secondary.main};
+          }
+          .inverted-tertiary {
+            color: ${theme.palette.tertiary.main};
+          }
+          .inverted-tertiary:hover::before {
+            background-color: ${theme.palette.tertiary.main};
+          }
+          .inverted-tertiary::after {
+            border-color: ${theme.palette.tertiary.main};
+          }
+          .inverted-white {
```

Comment:
```suggestion
          .inverted-black {
```
To maintain consistency of naming, I think we should continue using the `color` as `${color}` in `inverted-${color}`